src/tiled/createobjecttool.h

```diff
@@ -74,6 +76,7 @@ public slots:
     MapObjectItem *mNewMapObjectItem;
     MapObjectItem *mOverlayPolygonItem;
     Tile *mTile;
+    ObjectTemplate *mObjectTemplate;
```

Comment:
Why is this a member of `CreateObjectTool` rather than `CreateTemplateTool`?

The same question could be asked about `mTile` of course. That one is there for historical reasons and can be moved to `CreateTileObjectTool` since change 33e048b2ddc9722fbf491562fb9bc8883a34f78c.